tests/automember/test_automember.yml

```diff
@@ -0,0 +1,162 @@
+---
+- name: Test automember
+  hosts: ipaserver
+  become: true
+
+  tasks:
+
+  # CLEANUP TEST ITEMS
+
+  - name: Ensure group automember rule admins is absent
+    ipaautomember:
+      ipaadmin_password: SomeADMINpassword
+      name: admins
+      state: absent
+      type: group
+
+  - name: Ensure hostgroup automember rule ipaservers is absent
+    ipaautomember:
+      ipaadmin_password: SomeADMINpassword
+      name: ipaservers
+      state: absent
+      type: hostgroup
```

Comment:
Please, and a testing group and hostgroup (with `ipagroup` and `ipahostgroup`), so that the whole group or hostgroup can be cleaned up before and after test is executed. (`testgroup` and `testhostgroup` are great names, in this case.)

Reply:
Done